cartridges/app_storefront_training/cartridge/client/default/js/carousels/carousels.js

```diff
@@ -0,0 +1,40 @@
+require("../thirdyParty/slick-1.8.1/slick/slick");
+
+$(document).ready(function(){
+    $('.js-first-carousel').slick({
+        arrows: true,
+        centerMode: true,
+        centerPadding: '0px',
+        infinite: true,
+        slidesToShow: 1,
+        asNavFor: '.js-second-carousel'
+    });
+    $('.js-second-carousel').slick({
+        arrows: true,
+        dots: true,
+        centerMode: true,
+        centerPadding: '0px',
+        slidesToShow: 3,
+        slidesToScroll: 1,
+        infinite: true,
+        responsive: [
+            {
+              breakpoint: 1280,
+              settings: {
+                slidesToShow: 2,
+                slidesToScroll: 1,
+                centerMode: true
+              }
+            },
+            {
+              breakpoint: 900,
+              settings: {
+                slidesToShow: 1,
+                slidesToScroll: 1,
+                centerMode: true
+              }
+            }
+          ]
+    });
+    $('.item').css("display", "flex");
```

Comment:
Why are you applying this css here and not in a css file?

Reply:
I don´t remember exactly, but I think it was a conflict with the thirdParty CSS.